apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs

```diff
@@ -73,7 +76,7 @@ public BigtableVersion(DateTime timestamp)
                 nameof(timestamp),
                 $"The {nameof(BigtableVersion)} timestamp must be specified in UTC.");
 
-            Value = timestamp.Ticks / TicksPerMicro;
+            Value = (timestamp.Ticks - UnixEpoch.Ticks) / TicksPerMicro;
```

Comment:
It's not easy to tell, but presumably you've verified that you've caught all the places you use DateTime?

Reply:
Nope, I missed one. I'll add the test that should have broken.